Sources/MaliciousSiteProtection/Services/UpdateManager.swift

```diff
@@ -57,21 +78,34 @@ public struct UpdateManager: UpdateManaging {
         do {
             let request = DataKey.DataSet.APIRequest(threatKind: key.threatKind, revision: oldRevision)
             changeSet = try await apiClient.load(request)
+        } catch APIRequestV2.Error.urlSession(let error as URLError) {
+            Logger.updateManager.error("error fetching \(type(of: key)).\(key.threatKind): \(error)")
+            if dataSet.revision == 0 && error.code == .notConnectedToInternet {
```

Comment:
might be `APIRequestV2.Error.urlSession(URLError.notConnectedToInternet) where dataSet.revision == 0` for better readability